common/api-review/firestore.api.md

```diff
@@ -359,7 +359,7 @@ export abstract class QueryConstraint {
 }
 
 // @public
-export type QueryConstraintType = 'where' | 'orderBy' | 'limit' | 'limitToLast' | 'startAt' | 'startAfter' | 'endAt' | 'endBefore';
+export type QueryConstraintType = 'where' | 'orderBy' | 'limit' | 'limitToLast' | 'startAt' | 'startAfter' | 'endAt' | 'endBefore' | 'and' | 'or';
```

Comment:
given that we are not exposing 'and' and 'or' publicly, we should remove this update to the .md file

Reply:
Merging to a feature branch so that we can avoid exposing any public API changes.